state/protocol/badger/snapshot.go

```diff
@@ -295,12 +295,49 @@ func (s *Snapshot) Pending() ([]flow.Identifier, error) {
 	return s.pending(s.blockID)
```

Comment:
I think you could directly insert the business logic here: https://github.com/onflow/flow-go/blob/63052932c6c522989ab722ef931d2862001286ca/state/protocol/badger/snapshot.go#L337-L349

Reply:
never mind. My suggestion doesn't work, because the non-exported method actually takes a function argument (the blockID) while the exported method doesn't.